sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -53,7 +53,6 @@
     /**
      * Default UserGroup configuration.
      */
-    @NestedConfigurationProperty
```

Comment:
Why delete this?
For such kind of changes, could you please add more description in the PR description? Or add more description by `Start a review` by yourself.

![image](https://user-images.githubusercontent.com/13167207/158290240-40c41021-d242-4292-b74e-15d4132a48cc.png)

Reply:
for static inner classes, there's no need to add @NestedConfigurationProperty